WooCommerce/Classes/ViewRelated/Inbox/InboxViewModel.swift

```diff
@@ -169,8 +169,7 @@ private extension InboxViewModel {
     }
 
     func configureFirstPageLoad() {
-        // Listens only to the first emitted event.
-        onLoadTrigger.first()
+        onLoadTrigger
```

Comment:
Now that we reload every time the screen appears, I think we need to fix the unit tests.

Reply:
Good catch! I updated the unit test in 765ebad9836bd8caab2e3d5ac03ced3b5fb912b8.